frontend/src/pages/modelRegistry/screens/RegisterModel/RegisterModelErrors.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { Alert, AlertActionCloseButton, StackItem } from '@patternfly/react-core';
+
+type RegisterModelErrorProp = {
+  submitLabel: string;
+  submitError: Error;
+  errorName?: string;
+  versionName?: string;
+  modelName?: string;
+};
+
+const RegisterModelErrors: React.FC<RegisterModelErrorProp> = ({
+  submitLabel,
+  submitError,
+  errorName,
+  versionName = '',
+  modelName = '',
+}) => {
+  const [showAlert, setShowAlert] = React.useState<boolean>(true);
+
+  if (submitLabel === 'Register model' && errorName === 'modelVersion') {
```

Comment:
Looking closer actually @ppadti we probably shouldn't depend on specific string literals for logic like this in case those change later. Can we create enums for the possible values of `submitLabel` and `errorName` and use those instead?